whois.go

```diff
@@ -135,7 +135,15 @@ func (c Client) QueryAndParse(domain string) (*Response, error) {
 		key := strings.ToLower(strings.TrimSpace(line[:valueStartIndex]))
 		value := strings.TrimSpace(line[valueStartIndex+1:])
 		if response.ExpirationDate.Unix() != 0 && strings.Contains(key, "expir") && strings.Contains(key, "date") {
-			response.ExpirationDate, _ = time.Parse(time.RFC3339, strings.ToUpper(value))
+			switch {
+			case strings.HasSuffix(domain, ".br"):
+				response.ExpirationDate, _ = time.Parse("20060102", strings.ToUpper(value))
+			case strings.HasSuffix(domain, "co.ua"),
+				strings.HasSuffix(domain, "pp.ua"):
+				response.ExpirationDate, _ = time.Parse("02-Jan-2006 03:04:05 MST", strings.ToUpper(value))
+			default:
+				response.ExpirationDate, _ = time.Parse(time.RFC3339, strings.ToUpper(value))
+			}
```

Comment:
Any chance you could add a test in https://github.com/TwiN/whois/blob/ea4c6db71c1ce21647095a55c87846fc73a1065e/whois_test.go#L9 to confirm that it works as intended?

Reply:
I added the tests and for .br domains everything works fine. But I need help with `co.ua` and `pp.ua` domains.
The fact is that `domainExtension` for these domains is `ua` instead of `co.ua` and `pp.ua`. Because of this, requests are sent to `whois.ua` instead of `whois.co.ua` and `whois.pp.ua`.
Therefore, for these domains we receive `Invalid request.`
My programming skills are not very good to solve this problem qualitatively(

Reply:
Are you sure that it's not the opposite? I think it's because you wrongly added 
```
		"co.ua": "whois.co.ua",
		"pp.ua": "whois.pp.ua",
```
in the `referralWHOISServersCache`; you should try removing these two lines (which will cause the app to use whois.ua instead).

Reply:
Forget the above, I handled it in a separate commit